<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Zekyll Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="curRepoLabel">
        <property name="text">
         <string>Current repo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="curRepoCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="curRepoButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="defaultRepoButton">
        <property name="text">
         <string>Default repo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="gitCommit">
          <property name="text">
           <string>Git Commit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="gitReset">
          <property name="text">
           <string>Git Reset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="gitCheckout">
          <property name="text">
           <string>Git Checkout</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="gitStatus">
          <property name="text">
           <string>Git Status</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Zcode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyQLineEdit" name="zcode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Rev:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rev"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>File name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="file"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>User/repo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userRepo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Site:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="site">
        <item>
         <property name="text">
          <string>gh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gl</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>with index size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="indexSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sizeLeft">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sizeRight">
          <property name="text">
           <string>...</string>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Current index:</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="currentIndex">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="indexLeft">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="indexRight">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="minus">
        <property name="text">
         <string>â€“</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="up">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::UpArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="down">
        <property name="text">
         <string>...</string>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MyQTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Listing</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="MyQTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Section order listing</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="MyQTableWidget" name="tableWidget_2">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Inconsistent listing</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="messages_label">
      <property name="text">
       <string>Messages</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="messages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>tabWidget</zorder>
   <zorder>messages</zorder>
   <zorder>messages_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyQTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">myqtabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>MyQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
